lib/features/manage_account/domain/usecases/attempt_toggle_system_notification_setting_interactor.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:core/presentation/state/failure.dart';
+import 'package:core/presentation/state/success.dart';
+import 'package:dartz/dartz.dart';
+import 'package:tmail_ui_user/features/manage_account/domain/repository/notification_repository.dart';
+import 'package:tmail_ui_user/features/manage_account/domain/state/attempt_toggle_system_notification_setting_state.dart';
+
+class AttemptToggleSystemNotificationSettingInteractor {
+  final NotificationRepository _notificationRepository;
+  
+  AttemptToggleSystemNotificationSettingInteractor(this._notificationRepository);
+
+  Stream<Either<Failure, Success>> execute() async* {
+    try {
+      yield Right(AttemptingToggleSystemNotificationSetting());
+      await _notificationRepository.attemptToggleSystemNotificationSetting();
+      yield Right(AttemptToggleSystemNotificationSettingSuccess());
+    } on Exception catch (_) {
```

Comment:
What happens if it throws an instance not an exception?

Reply:
For notification, system setting & cache setting will always be synced as soon as https://github.com/linagora/tmail-flutter/issues/2881 is implemented

Reply:
Cache temporarily turned off

Reply:
`on Exception` removed